core/src/main/java/com/scalar/db/transaction/consensuscommit/RollbackMutationComposer.java

```diff
@@ -59,10 +59,6 @@ public void add(Operation base, @Nullable TransactionResult result) throws Execu
       // the record was not prepared (yet) by this transaction or has already been rolled back
       return;
     }
-    if (latest.isCommitted()) {
-      // the record has already been rolled back
-      return;
-    }
```

Comment:
Removed this because when we reach here, we decide to abort the transaction, so it's impossible that the transaction ID is the same between the latest record (`latest`) and `result` but the latest record is committed.